### PR TITLE
DEPLOY-530: Security vulnerability in Alfresco/alfresco-anaxes-hello-world-ui

### PR DESCRIPTION
**Pull request by cristina-sirbu**
The exciting and shiny new Hello World UI.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,191 +1,21 @@
...
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
\ No newline at end of file
+MIT License
+
+Copyright (c) 2017 Denys Vuika
+
```
Comment (kconkas) [inline LICENSE]: Why did we change the license for this project from Apache to MIT and why is this project's copyright owned by Denys Vuika?

File: src/app.config.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
+{
+    "$schema": "../node_modules/@alfresco/adf-core/app.config.schema.json",
+    "ecmHost": "http://localhost",
+    "bpmHost": "http://localhost",
+    "backendHost": "http://localhost:8080",
```
Comment (kconkas) [inline src/app.config.json]: I find this naming of configuration variables rather confusing: we are naming variables `<something>Host`, yet they actually hold URLs.

File: src/app.config.json [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    
+    "providers" : "ALL",
+    "application": {
+        "name": "Alfresco ADF Appplication"
```
Comment (kconkas) [inline src/app.config.json]: Do we really want to stick with "Alfresco ADF Application" as our hello-world app name?

**Review by kconkas — CHANGES_REQUESTED** [review]
Overall excellent job. Minor changes may be appropriate (please see my comments).

File: LICENSE (hunk 1 above)
Comment (cristina-sirbu) [reply LICENSE]: This is the new license that gets generated using [Yeoman Generator Angular 2 Alfresco Application](https://github.com/Alfresco/generator-ng2-alfresco-app) and it states: ```The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.```

File: src/app.config.json (hunk 2 above)
Comment (cristina-sirbu) [reply src/app.config.json]: I didn't name those for ACS and APS. They are generated. I created just the one for backend service and I tried to follow the pattern.

File: src/app.config.json (hunk 3 above)
Comment (cristina-sirbu) [reply src/app.config.json]: Forgot to change it. Thank you!

**Review by kconkas — APPROVED** [review]
LGTM but let's discuss the issue of this project from now on by default being under MIT license and copyrighted by an individual Alfresco employee (unrelated to this project) before merging.

File: LICENSE (hunk 1 above)
Comment (rgauss) [reply LICENSE]: We should keep the license as Apache and copyright as Alfresco Software, Inc.

**Comment by ehardon** [comment]
LGTM! 👍

File: package.json [hunk 4]
```diff
@@ -1,60 +1,81 @@
 {
-  "name": "adf-app",
-  "version": "0.0.1",
-  "license": "",
+  "name": "adf-cli-acs-aps-template",
+  "version": "2.6.0",
+  "blueprint": "Process and Content Services",
+  "description": "ACS APS ADF Application with Angular CLI",
+  "license": "MIT",
```
Comment (rgauss) [inline package.json]: Probably need Apache here as well.

**Review by rgauss — CHANGES_REQUESTED** [review]
Minor license cleanup

File: package.json (hunk 4 above)
Comment (cristina-sirbu) [reply package.json]: Thank you! Changed it in my last commit.

**Review by rgauss — APPROVED** [review]

**Pull request merged**